types/node/globals.d.ts

```diff
@@ -62,6 +62,9 @@ interface AbortSignal {
      * Returns true if this AbortSignal's AbortController has signaled to abort, and false otherwise.
      */
     readonly aborted: boolean;
+
+    onabort: ((this: AbortSignal, event: Event) => any) | null;
+    addEventListener(type: 'abort', listener: (this: AbortSignal, event: Event) => any, options?: { once?: boolean }): void;
```

Comment:
We should probably move the `once` to a shared place so it can be re-used, probably `events.d.ts`.

Reply:
sure, can i add an events namespace import to the globals.d.ts or is that an issue?